### PR TITLE
[pr2eus] add volume key arg for robot-interface :speak

### PR DESCRIPTION
**Pull request by knorth55**
add `:volume` argument in `robot-interface` `:speak` methods.

### CODE REVIEW

**Comment by knorth55** [comment]
@708yamaguchi can you review this PR?

**Comment by 708yamaguchi** [comment]
This PR seems to work correctly on Fetch15 robot. :100:

File: pr2eus/robot-interface.l [hunk 1]
```diff
@@ -1236,17 +1236,18 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
 ;;
 ;; for text-to-speech services
 (defmethod robot-interface
-  (:play-sound (sound &key arg2 (topic-name "robotsound") wait)
-   (funcall #'play-sound sound :arg2 arg2 :topic-name topic-name :wait wait))
-  (:speak (text &key (lang "") (topic-name "robotsound") wait)
+  (:play-sound (sound &key arg2 (topic-name "robotsound") wait (volume 1.0))
```
Comment (k-okada) [inline pr2eus/robot-interface.l:1239]: how about `(:play-sound (sound &rest args &allow-other-keys) (apply #'playsound sound args)(` or `(:play-sound (&rest args) (apply #'play-sound args))` ?

Comment (knorth55) [reply pr2eus/robot-interface.l]: thank you for your review.
I update to pass all the args to `play-sound` with `apply`

**Comment by knorth55** [comment]
I tested with PR2 and it works.

**Comment by 708yamaguchi** [comment]
I was able to call
```lisp
(send *ri* :play-sound 1 :volume 0.5)
(send *ri* :play-sound 1 :volume 1.0)
(send *ri* :play-sound 1 :volume 2.0)
```
with Fetch15.

It seems to work correctly.

**Comment by knorth55** [comment]
@k-okada travis passed! can you merge this PR?

**Comment by shmpwk** [comment]
@k-okada can you please merge this PR? I need this PR for baxter demo.

**Review by k-okada — APPROVED** [review]

**Pull request merged**